site/docs/components/_content-template/index.mdx

```diff
@@ -0,0 +1,218 @@
+  usage.
+
+  For screenshots use Markdown:
+
+  ![Text alternative for screenshot](./path/to/screenshot.jpg)
+
+  For live demos in Storybook use:
+
+  <Canvas>
+    <Story id="..." />
+  </Canvas>
+-->
+
+<!--
+  Introductory text that briefly explains the purpose of the component and
+  anything else of note.
+-->
+
+## When to use Component Name
+
+<!-- Bulleted list of scenarios in which this component can be used  -->
+
+- ...
+- ...
+
+## When not to use Component Name
+
+<!--
+  Bulleted list of scenarios where this component is not appropriate. Where
+  possible, include a link to an alternative component that should be used in
+  it's place
+-->
+
+- ...
+- ...
+
+## Configurations
+
+<!--
+  A visual list of possible configurations of the component.
+
+  The purpose is to convey the main features and/or usage options of the
+  component (e.g. the Data Grid's striped rows might be a "configuration" to
+  showcase here). The configurations listed here should aim to be comprehensive,
+  but not necessarily exhaustive (some complex components may have vast numbers
+  of possible permutations, so listing out each one would be impossible and
+  overwhelming to readers).
+
+  Each confguration should have its own section following the structure outlined
+  below:
+-->
+
+### Configuration title (e.g. Striped rows)
+
+<!--
+  A brief description of what this configuration provides to users and when it
+  should be used.
+-->
+
+<!--
+  A live example of the component configuration, with access to the correspoding
+  source code.
+
+  In Storybook this is done using something like:
+
+  <Canvas>
+    <Story id="..." />
+  </Canvas>
+-->
+
+## Content
+
+<!--
+  OPTIONAL: Remove this section if it is not applicable.
+
+  Add any component-specific content guidance here, use visual examples where
+  appropriate.
+
+  For example, a Button component might include guidance on recommended terms to
+  use for the label text, recommended maximum character counts, etc.
+-->
+
+## Accessibility
+
+<!--
+  OPTIONAL: Remove this section if it is not applicable.
+
+  Describe any considerations the users may need to take into account when using
+  the component.
+-->
+
+## References
+
+<!--
+  OPTIONAL: Remove this section if it is not applicable.
+
+  If there are industry standards for this component, list them in this
+  section. Include links out to useful articles.
+-->
+
+- [Relevant industry standard](https://example-standard.org/)
+- [Useful article](https://some-site.com/article/bla)
+- ...
+
+## Designing
+
+### Theming considerations
+
+<!--
+  Describe how Salt characteristics and theming impact the component, and any
+  modifications that can be made.
+-->
+
+### Design guidance
+
+<!--
+  Add a bulleted list of any other design-related guidance the user may need to
+  consider.
+-->
+
+### Additional materials
+
+<!--
+  Add a bulleted list of links to additional, component-specific materials:
```

Comment:
May have to differentiate between internal vs external user and link each consumer group out to relevant docs?

Reply:
We're quite a way off having the technical means to add JPMC-only content on the site, so I'd suggest that this needs to be limited to only public links for now (but I can add a note in the guidance to that effect).

Reply:
One for @bhoppers2008 to go into more detail on, I believe we don't share this info out in the open as it is more a working file for us as designers to re-design and re-build our components whilst also producing a spec for handoff to devs and it may contain sensitive data that's only relevant for JPMC designers.